contexts/nft-context.js

```diff
@@ -4,19 +4,18 @@ import { useWeb3React } from '@web3-react/core'
 import detectEthereumProvider from '@metamask/detect-provider'
 import Web3 from 'web3'
 
-import { CONTRACTS, C_CHAIN_ID } from 'config'
+import { CONTRACTS } from 'config'
 import getNFTABI from 'libs/abis/nft'
 import GOVERNANCE_ABI from 'libs/abis/governance.json'
 import { usePopup } from 'contexts/popup-context'
 
 const ContractContext = createContext(null)
 
 export function NFTContractProvider({ children }) {
-  const { account, library, chainId } = useWeb3React();
+  const { account, library } = useWeb3React();
   const { setPopUp } = usePopup();
   const [loading, setLoading] = useState(false);
 
-  const isWrongNetwork = useMemo(() => chainId !== C_CHAIN_ID, [chainId])
```

Comment:
Why are we removing the network check here?

We have plenty of evidence from users that this is a useful feature.

Reply:
We have already had a network check feature in contract-context.
We can use it anywhere. So I have removed it in nft-context.

Reply:
https://github.com/Snowball-Finance/app-v2/blob/2ae1ed0403391c5688c653118c954e519a0ddc88/contexts/contract-context.js#L17